neurokit2/signal/signal_psd.py

```diff
@@ -5,7 +5,19 @@
 
 
 def signal_psd(
-    signal, sampling_rate=1000, method="welch", show=False, normalization=False, min_frequency=0, max_frequency=np.inf, window=None, window_type='hann', ar_order=16, order_criteria="KIC", order_corrected=True, **kwargs
+    signal,
+    sampling_rate=1000,
+    method="welch",
+    show=False,
+    normalization=True,
+    min_frequency=0,
+    max_frequency=np.inf,
+    window=None,
+    window_type='hann',
+    ar_order=16,
```

Comment:
I'd rename `ar_order` by `order` (so it's consistent with the following args) and mention that it's used for autoregressive methods